client/state/lasagna/actions.js

```diff
@@ -0,0 +1,2 @@
+export const socketConnected = () => ( { type: 'LASAGNA_SOCKET_CONNECTED' } );
+export const socketDisconnected = () => ( { type: 'LASAGNA_SOCKET_DISCONNECTED' } );
```

Comment:
The action types are added to `client/state/action-types` but plain strings are used here.

Reply:
That should be fixed in the next PR https://github.com/Automattic/wp-calypso/pull/40720